app/src/main/java/com/procrastimax/birthdaybuddy/MainActivity.kt

```diff
@@ -57,7 +57,7 @@ class MainActivity : AppCompatActivity() {
         }
 
         val ft: FragmentTransaction = supportFragmentManager.beginTransaction()
-        ft.add(
+        ft.replace(
```

Comment:
I think the `replace` doesn't solve the real problem, and because the `add` save the previous fragment state so I think there should be another way to solve the overlapped text.

Reply:
This seems to fix it.
Thank you!